Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Cursors/CursorModifier.cs

```diff
@@ -164,7 +181,8 @@ public Vector3 GetModifiedPosition(IMixedRealityCursor cursor)
             }
 
             FocusDetails focusDetails;
-            if (MixedRealityToolkit.InputSystem != null && MixedRealityToolkit.InputSystem.FocusProvider.TryGetFocusDetails(cursor.Pointer, out focusDetails))
+            if (InputSystem?.FocusProvider != null && 
```

Comment:
>InputSystem?.FocusProvider != null &&  [](start = 16, length = 38)

This seems slightly different compared to the past (i.e. there wasn't a FocusProvider null check) - was this intentional to add this focus provider null check? Is that a side effect of all this work?

Reply:
it was intentional. we would hit a null reference if the focus provider were ever not setup